BootloaderCommonPkg/Library/IppCryptoLib/sha256.c

```diff
@@ -16,8 +16,18 @@
 
 Ipp8u* Sha256 (const Ipp8u* pMsg, Ipp32u msgLen, Ipp8u* pMD)
 {
-  ippsHashMessage_rmf(pMsg, msgLen, pMD, ippsHashMethod_SHA256 ());
-  return pMD;
+  #if (FixedPcdGet8(PcdIppHashLibSupported) & 0x2)
```

Comment:
You don't have to use #if, #else, #endif if you use FixedPcd. Just use if (FixedPcd()) {} else {}. Compiler will optimize this by FixedPcd value.

Reply:
Compiler still tries to execute code in if (FixedPcd()) {}  for False case too. Checked with creating an error case.
Let me know if i am missing anything here.